lib/wibox/init.lua

```diff
@@ -310,8 +310,9 @@ local function new(args)
         ret._drawable:_inform_visible(w.visible)
     end)
 
+    --TODO v5 deprecate this and use `wibox.object`.
     for k, v in pairs(wibox) do
-        if type(v) == "function" then
+        if (not rawget(ret, k)) and type(v) == "function" then
```

Comment:
I'm curious: Which functions are overwritten here? In my local git checkout (which is not up to date), this change makes no difference.

Reply:
`gears.object.connect_signal` (method) was replaced by `wibox.connect_signal` (global class level function)